packages/contracts/docs/osx/02-how-to-guides/02-plugin-development/07-publication/02-metadata.md

```diff
@@ -0,0 +1,122 @@
+
+The intention is to provide an appealing overview of each releases functionality.
+It can be updated with each call to [`createVersion()`](../../../03-reference-guide/framework/plugin/repo/IPluginRepo.md/#external-function-createversion) in `IPluginRepo` by the repo maintainer
+It can be replaced at any time with [`updateReleaseMetadata()`](../../../03-reference-guide/framework/plugin/repo/IPluginRepo.md/#external-function-updatereleasemetadata) in `IPluginRepo` by the repo maintainer
+
+The `release-metadata.json` file consists of the following entries:
+
+| Key         | Type        | Description                                                                  |
+| ----------- | ----------- | ---------------------------------------------------------------------------- |
+| name        | `string`    | Name of the plugin (e.g. `"Multisig"`)                                       |
+| description | `string`    | Description of the plugin release and its functionality.                     |
+| images      | UNSPECIFIED | Optional. Contains a series of images advertising the plugins functionality. |
+
+#### Example
+
+```json
+{
+  "name": "Multisig",
+  "description": "",
+  "images": {}
+}
+```
+
+### Build Metadata
+
+The build metadata is a `.json` file stored on IPFS with its IPFS CID published for each build **only once** in the [PluginRepo](../../../01-how-it-works/02-framework/02-plugin-management/01-plugin-repo/index.md) (see also the section about [versioning](../07-publication/01-versioning.md/#)).
+
+The intention is to inform about the changes that were introduced in this build compared to the previous one and give instructions to the App frontend and other users on how to interact with the plugin setup and implementation contract.
+It can be published **only once** with the call to [`createVersion()`](../../../03-reference-guide/framework/plugin/repo/IPluginRepo.md/#external-function-createversion) in `IPluginRepo` by the repo maintainer
+
+| Key         | Type        | Description                                                                                               |
+| ----------- | ----------- | --------------------------------------------------------------------------------------------------------- |
+| ui          | UNSPECIFIED | A special formatted object containing instructions for the App frontend on how to render the plugin's UI. |
+| change      | `string`    | Description of the code and UI changes compared to the previous build of the same release.                |
+| pluginSetup | `object`    | Optional. Contains a series of images advertising the plugins functionality.                              |
+
+Each build metadata contains the following field
```

Comment:
"following fields:"

Reply:
Changed in defce3f5d16cac6b573476b5c5a68d86f26597b6